src/main/java/ch/njol/skript/lang/LoopSection.java

```diff
@@ -0,0 +1,52 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.lang;
+
+import ch.njol.skript.sections.SecLoop;
+import ch.njol.skript.sections.SecWhile;
+import org.bukkit.event.Event;
+
+import javax.annotation.Nullable;
+
+/**
+ * Represents a loop section.
+ * 
+ * @see ch.njol.skript.sections.SecWhile
+ * @see ch.njol.skript.sections.SecLoop
```

Comment:
`@see` is deprecated, can we change these to a `{@link}`?

Reply:
Where did you get that from? Not found on https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#@see or https://docs.oracle.com/en/java/javase/17/docs/specs/javadoc/doc-comment-spec.html